articles/quickstart/spa/angular/interactive.md

```diff
@@ -11,6 +11,10 @@ github:
   path: Sample-01
 ---
 
+:::note
+For a deep dive into implementing user authentication in Angular, visit the [Angular Authentication By Example](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication). This guide provides information for additional tasks, like creating a signup button.
+:::
```

Comment:
Yes, I'm still wondering where the best place to add it would be. Dan requested the notes be moved to the top of the page. A good place is before content starts or right after the intro before the TOC.